x=5
print(x)
del x
#print(a) error but already delete a